python/lsst/sims/utils/coordinateTransformations.py

```diff
@@ -441,16 +633,21 @@ def getRotTelPos(raRad, decRad, longRad, latRad, mjd, rotSkyRad):
 
     @param [out] rotTelPos in radians.
 
-    WARNING: as of 13 April 2015, this method does not agree with OpSim on
-    the relationship between rotSkyPos and rotTelPos
+    WARNING: As of 13 April 2015, this method does not agree with OpSim on
+    the relationship between rotSkyPos and rotTelPos.  This is due to a
+    discrepancy between the time that OpSim uses as the MJD when calculating
+    rotTelPos and the time that OpSim reports as being the actual expmjd
+    of the exposure (one is the beginning of the exposure; the
+    other is the middle of the exposure).
```

Comment:
I assume the MJD is mid-exposure and rotTelPos is the start of the exposure...but it might be good to say so explicitly and tell someone to go update the OpSim documentation so at least it's recorded somewhere.